poly-commitment/src/evaluation_proof.rs

```diff
@@ -419,6 +446,7 @@ pub struct OpeningProof<G: AffineCurve> {
     pub z1: G::ScalarField,
     #[serde_as(as = "o1_utils::serialization::SerdeAs")]
     pub z2: G::ScalarField,
+    /// A final folded commitment base
```

Comment:
Folded?

Reply:
Yeah, but not in the instance-folding way. At each (1/log N) step of bulletproofs we have several vectors that get "folded in half", so that's what I meant. I can say "log-folded" or "IPA-folded" if this is more clear?

Reply:
ah yeah, much more